stdlib/asm/crypto/stark/random_coin.masm

```diff
@@ -651,7 +518,6 @@ end
 #! [0, lde_size] and store it in memory starting from `query_ptr`.
 #! The list is stored as `(r, depth, y, y)` where `depth` is `log(lde_domain_size)`.
 #!`depth` is needed when computing the deep queries.
-#! TODO: the case of duplicate queries
 #!
```

Comment:
Are we handling this case now?

Reply:
This should've been removed a while back when we removed the deduplication check in Winterfell here https://github.com/facebook/winterfell/issues/216